lib/async_job/async_job.ex

```diff
@@ -160,6 +160,12 @@ defmodule Antikythera.AsyncJob do
 
   During testing, the rate limiting feature may disrupt smooth test executions.
   To bypass rate limiting in your tests you can use `Antikythera.Test.AsyncJobHelper.reset_rate_limit_status/1`.
+
+  ### Payload
+
+  Payloads are registered by calling `register/3` and used in `run/3`, `abandon/3` or `inspect_payload/1`.
+  Payloads are compressed as binary when `register/3` is called and kept in memory until job executions, so large payloads could degrade overall system performance.
+  Please avoid including large data in payloads and fetch large data in `run/3` if you can.
```

Comment:
Let's slightly clarify the alternative of "putting large data in payload": 

Please avoid including large data in payloads and fetch large data in `run/3` if you can.
=>
Please avoid including large data in payloads; instead put ID of the data in payload and fetch the whole data in `run/3`.

Reply:
Indeed it's confusing. Fixed in 442c876a0b44b45e4b7117977b1121a37b83906c